src/grisp_pins.erl

```diff
@@ -0,0 +1,64 @@
+-module(grisp_pins).
```

Comment:
Does it make sense to make this file a header file instead? That way we wouldn't have to use the `import` directive and the function would be local to any module including them leading to some possible compiler optimizations (such as inlining).

Reply:
Are functions in headers fine? Or do you want to create records containing functions?

Reply:
Functions in header files works. Header files in Erlang are very simple, the contents is just copied directly into the module including it before compilation.

However, considering the performance point made below, I'd say let's keep it like this. No point in making it fancy or complicated if it is not a performance bottleneck. Readability FTW.